glTFLoader/Interface.cs

```diff
@@ -129,6 +236,52 @@ public static void SaveModel(Gltf model, Stream stream)
             }
         }
 
+        public static void SaveBinaryModel(this Gltf model, byte[] buffer, string filePath)
+        {
+            using (Stream stream = File.Create(filePath))
+            {
+                SaveBinaryModel(model, buffer, stream);
+            }
+        }
+
+        public static void SaveBinaryModel(this Gltf model, byte[] buffer, Stream stream)
+        {
+            using (var wb = new BinaryWriter(stream))
+            {
+                SaveBinaryModel(model, buffer, wb);
+            }
+        }
+
+        public static void SaveBinaryModel(this Gltf model, byte[] buffer, BinaryWriter binaryWriter)

```

Comment:
Consider allowing buffer to be null which will then not write out the BIN chunk. BIN chunk is not always required for all glTF files.